tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -116,8 +116,6 @@ def start(self):
 
     def stop(self):
         self._logger.info("Stopping all traffic generation")
-        self._producer.stop()
-        self._consumer.stop()
         self._producer.wait()
         self._logger.info("Producer stopped")
```

Comment:
> wait() after stop() does not make sense. My intuition is that the test
author intended to only run wait() instead of hard stopping the
operation so I remove the calls to stop() as they are likely erroneous.

Does this theory still hold up for `test_throughput_simple` in this same file? in that test it kinda looks like it depends on the test being stopped as if the traffic generator doesn't otherwise have a natural stop condition?

Reply:
yikes, only stop is needed here